travvy/src/components/Onboarding.vue

```diff
@@ -142,6 +157,32 @@ export default {
         console.log(error)
         }
     },
+     
+    // async addUserType(){
+    //   console.log(this.email)
+    //   //console.log(this.preferred_activity_type)
+      
+      
+    //   //console.log(this.preferred_activity_type)
+    //   console.log(this.errors)
+    //    try {
+
+    //      // if this. prefered act type == fam, act id = 116 ... 
+
+    //     const response = await AuthenticationService.addUserType({
+    //       "Email": this.email,
+    //       "activity_id": 116
+          
+    //       })
+    //     console.log(response);
+       
+
+    //   } catch (error) {
+    //     console.log(error)
+    //     }
+     
+
+    // },
```

Comment:
Could these comments be deleted for cleaner code? Or are they necessary?

Reply:
Great catch - will edit this.